onnxscript/tests/function_libs/torch_lib/ops_test_data.py

```diff
@@ -870,6 +870,7 @@ def _where_input_wrangler(
         reason="fixme: ORT only supports float32 when accumulate is True:  MLFloat16 data type is not supported with ScatterND when reduction is 'add'",
     ),
     TorchLibOpInfo("ops.aten.index_put", core_ops.aten_index_put),
+    TorchLibOpInfo("ops.aten._unsafe_index_put", core_ops.aten__unsafe_index_put),
```

Comment:
This op does not exist in extra_opinfo.py, so it will need to be created. Simply add this entry to the file

```
    opinfo_core.OpInfo(
        "ops.aten._unsafe_index_put",
        aten_name="_unsafe_index_put",
        dtypes=common_dtype.floating_types(),
        sample_inputs_func=sample_inputs_index_put,
        supports_out=False,
    ),
```

will do

Reply:
I did but I don't really know what I'm doing so if you have a PR I could use as an example.

Reply:
Sure, https://github.com/microsoft/onnxscript/pull/1260/files and https://github.com/microsoft/onnxscript/pull/1284 are good examples.